cypress/fixtures/fireboltCalls/authentication.js

```diff
@@ -0,0 +1,182 @@
+                description: 'Validation of the Authentication Token partner Format',
+              },
+            ],
+          },
+        ],
+      },
+      DECODE_BASE64_AUTHENTICATION_TOKEN: {
+        data: [
+          {
+            type: 'decode',
+            specialCase: 'base64',
+            validations: [
+              {
+                field: 'issueDate',
+                mode: 'regex',
+                format: 'TOKEN_REGEXP',
+                type: '[0-9]{4}-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])T([01][0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9])',
+                description: 'Validation of the Authentication Token issueDate Format',
+              },
+            ],
+          },
+          {
+            type: 'decode',
+            specialCase: 'base64',
+            validations: [
+              {
+                field: 'notOnOrAfter',
+                mode: 'regex',
+                format: 'TOKEN_REGEXP',
+                type: '[0-9]{4}-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])T([01][0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9])',
+                description: 'Validation of the Authentication Token notOnOrAfter Format',
+              },
+            ],
+          },
+        ],
+      },
+      DECODE_BASE64_AUTHENTICATION_DEVICE_TOKEN: {
+        data: [
+          {
+            type: 'decode',
+            specialCase: 'base64',
+            validations: [
+              {
+                field: 'issueDate',
+                mode: 'regex',
+                format: 'TOKEN_REGEXP',
+                type: '[0-9]{4}-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])T([01][0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9])',
+                description: 'Validation of the Authentication Token issueDate Format',
+              },
+            ],
+          },
+        ],
+      },
+    },
+  },
+};
+
+exports.GET_THE_AUTHENTICATION_TOKEN_FOR_PLATFORM = {
+  method: 'authentication.token',
+  params: { type: 'platform' },
+};
+
+exports.PLATFORM_AUTHENTICATION_TOKEN = {
+  method: 'authentication.token',
+  validationJsonPath: 'result',
+  content: this.AUTHENTICATION_VARIABLES.DEFAULTS.AUTHENTICATION.DECODE_JWT_AUTHENTICATION_TOKEN,
+};
+
+exports.GET_THE_AUTHENTICATION_TOKEN_FOR_DEVICE = {
+  method: 'authentication.token',
+  params: { type: 'device' },
+};
+
+exports.DECODE_BASE64_AUTHENTICATION_TOKEN = {
+  method: 'authentication.token',
+  validationJsonPath: 'result',
+  content: this.AUTHENTICATION_VARIABLES.DEFAULTS.AUTHENTICATION.DECODE_BASE64_AUTHENTICATION_TOKEN,
+};
+
+exports.GET_THE_AUTHENTICATION_TOKEN_FOR_DISTRIBUTOR = {
+  method: 'authentication.token',
+  params: { type: 'distributor' },
+};
+
+exports.GET_THE_AUTHENTICATION_TOKEN_FOR_DISTRIBUTOR_CIMA = {
+  method: 'authentication.token',
+  params: { type: 'distributor', options: { clientId: 'CIMA' } },
+};
+
+exports.GET_THE_AUTHENTICATION_TOKEN_FOR_DISTRIBUTOR_OAT = {
+  method: 'authentication.token',
+  params: { type: 'distributor', options: { clientId: 'OAT' } },
+};
```

Comment:
are we supposed to keep direct object as params here ?

Reply:
we can either pass parameters directly as objects (which is the expected) or create a separate js file to store parameter values, similar to how it's done in modules/<moduleName>.json. we need to import this file in fireboltCalls/<moduleName>.js to utilize these parameters.